EvoS.Framework/Network/Static/AuthInfo.cs

```diff
@@ -15,7 +15,7 @@ public class AuthInfo
         public string Handle { get; set; }
         public string Password { internal get; set; }  // internal so that it is not serialized
         public long SteamId { get; set; }
-        public string TicketData { internal get; set; }  // internal so that it is not serialized
+        public string TicketData { get; set; }
```

Comment:
Why do you want to log them?

Reply:
i needed to read the data but i changed it back to interntal and created a method to get the data now